server/api.go

```diff
@@ -245,7 +211,6 @@ func (p *Plugin) handleEndCall(w http.ResponseWriter, r *http.Request, channelID
 		return
 	}
 
-	p.metrics.IncWebSocketEvent("out", "call_end")
```

Comment:
I believe so, since I refactored the call into a wrapper we now send those metrics in one place only.

https://github.com/mattermost/mattermost-plugin-calls/blob/e34a56cb03085e0e3318bb91fae93a4fd45f0816/server/websocket.go#L64